modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -87,80 +87,300 @@ $ docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig registr
+I0829 13:25:21.569345       1 node.go:44] Environment information: kernel version 4.18.0-305.10.2.rt7.83.el8_4.x86_64
+I0829 13:25:21.569367       1 main.go:53] Running the oslat command with arguments [--duration 600 --rtprio 1 --cpu-list 4,6,52,54,56,58 --cpu-main-thread 2] <1>
+I0829 13:35:22.632263       1 main.go:59] Succeeded to run the oslat command: oslat V 2.00
+Total runtime: 		600 seconds
 Thread priority: 	SCHED_FIFO:1
-CPU list: 		3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50
+CPU list: 		4,6,52,54,56,58
 CPU for main thread: 	2
 Workload: 		no
 Workload mem: 		0 (KiB)
-Preheat cores: 		48
+Preheat cores: 		6
 
 Pre-heat for 1 seconds...
 Test starts...
 Test completed.
 
-Core: 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50
-CPU Freq: 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2092 2096 2096 2096 2092 2092 2096 2096 2096 2096 2096 2096 2096 2096 2096 2096 2092 2096 2096 2092 2096 2096 2096 2096 2092 2096 2096 2096 2092 2096 2096 2096 2096 2096 2096 (Mhz)
-...
-Maximum: 3 4 3 3 3 3 3 3 4 3 3 3 3 4 3 3 3 3 3 4 3 3 3 3 3 3 3 3 3 4 3 3 3 3 3 3 3 4 3 3 3 3 3 4 3 3 3 4 (us)
-----
+        Core:	 4 6 52 54 56 58
+    CPU Freq:	 2096 2096 2096 2096 2096 2096 (Mhz)
+    001 (us):	 19390720316 19141129810 20265099129 20280959461 19391991159 19119877333
+    002 (us):	 5304 5249 5777 5947 6829 4971
+    003 (us):	 28 14 434 47 208 21
+    004 (us):	 1388 853 123568 152817 5576 0
+    005 (us):	 207850 223544 103827 91812 227236 231563
+    006 (us):	 60770 122038 277581 323120 122633 122357
+    007 (us):	 280023 223992 63016 25896 214194 218395
+    008 (us):	 40604 25152 24368 4264 24440 25115
+    009 (us):	 6858 3065 5815 810 3286 2116
+    010 (us):	 1947 936 1452 151 474 361
+  ...
+     Minimum:	 1 1 1 1 1 1 (us)
+     Average:	 1.000 1.000 1.000 1.000 1.000 1.000 (us)
+     Maximum:	 37 38 49 28 28 19 (us) <3>
+     Max-Min:	 36 37 48 27 27 18 (us)
+    Duration:	 599.667 599.667 599.667 599.667 599.667 599.667 (sec)
+----
+<1> The list of CPUs running the `oslat` command. Seven CPUs are provided through the `LATENCY_TEST_CPUS` variable. Only six CPUs are displayed in total because one is used to run the `oslat` tool.
+<2> The user is notified about the measured latency and the configured latency.
+<3> The maximum latency values in microseconds that are measured on each CPU.
+
 
 [id="cnf-performing-end-to-end-tests-image-parameters_{context}"]
 == Image parameters
```

Comment:
While reading the draft, this subtitle looked very weird. It was as if the latency tests could use different images. The "chapter" separation here is non-obvious.

Reply:
Unfortunately, this is a tool limitation. We can't control how the different heading levels render.

Reply:
What about making Running latency tests a higher level section? It deserves it as it is a very specific testing case and it is big enough too. You can then move Latency testing right  before the Disconnected mode I think.

Reply:
I have to look into if moving Running latency tests to a higher level would break any links. 

`You can then move Latency testing right before the Disconnected mode I think.`
Do you mean Discovery mode section? Also, I'm not familiar with end-to-end testing, would that be a more logical place for running the latency tests?

Reply:
Nope, I really meant the Disconnected section: https://deploy-preview-36024--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes?utm_source=github&utm_campaign=bot_dp#cnf-performing-end-to-end-tests-disconnected-mode_cnf-master

The issue right now is that Latency testing does not even show in the TOC.

Reply:
Ah, it does.. nevermind. However all the sections around talk about end to end tests and the way to execute them.. and sadly most of that content does not apply to latency testing except discovery mode, disconnected mode and single node.

So I wonder if the current progression of sections is the best one


    Prerequisites
    Running the latency tests
    Image parameters
    Dry run
    Disconnected mode
    Discovery mode
    Running in a single node cluster
    Troubleshooting
    Test reports
    Impacts on the cluster

Reply:
I updated the headings after your comment, that's why the latency test shows up. I can make another quick update and move that section before Disconnected mode

Reply:
I would probably propose something like this


    Prerequisites
    Dry run
    Disconnected mode
    Running in a single node cluster
    (all above applies to test environment)

    Impacts on the cluster -- the title is misleading, it also describes what tests are doing what
          Image parameters - This one is kind of weird, it should probably belong to the Impacts on the cluster  as it describes configuration for the described tests
    Discovery mode
    Running the latency tests
    (all are about tests and about the impact to the cluster)

    Troubleshooting
    Test reports
    (what to do with the results and when something goes wrong)

Reply:
@fedepaol I think I would like to hear your opinion here, the section titles are a bit of a mess

Reply:
Let me get back to you about this!

Reply:
@MarSik Thank you for your patience, Martin! I updated the document with the proposed section order.  However, I'd also like to get it reviewed by the Perf/Scale team, just to make sure that I'm not stepping on any toes with the restructuring.

In the meantime, you can check it in the [preview](https://deploy-preview-36024--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes?utm_source=github&utm_campaign=bot_dp#cnf-performing-end-to-end-tests-running-the-latency_tests). Let me know what you think!

Reply:
I'd rename image parameters to "Override images used by tests" if you want, but I don't feel it should be under the other section. It's totally unrelated and explains how to override the images used for pods.

Reply:
@MarSik @fedepaol 
If we put Image parameters as a subsection, we'll have to adjust the next subsection under that. In that case, we'll have the same problem as previously with the different test sections, where the headings didn't stand out enough.

I'd propose to only rename the headings:
- Impacts on the cluster > Impacts of tests on the cluster
- Image parameters > Override test image parameters

Let me know if I can proceed with these changes.